dynamic_rest/viewsets.py

```diff
@@ -468,10 +485,45 @@ def create(self, request, *args, **kwargs):
             {"name": "Lucky", "age": 3}
         ]
         """
-        plural_name = self.get_serializer_class().get_plural_name()
-        if isinstance(request.data, list):
-            return self._create_many(request.data)
-        elif plural_name in request.data and len(request.data) == 1:
-            return self._create_many(request.data[plural_name])
+        if self.is_bulk_request(request):
+            return self._create_many(self._get_bulk_payload(request))
         return super(DynamicModelViewSet, self).create(
             request, *args, **kwargs)
+
+    def _destroy_many(self, data):
+        for instance in self.get_queryset().filter(
+            id__in=[d['id'] for d in data]
+        ):
+            self.check_object_permissions(self.request, instance)
+            self.perform_destroy(instance)
+        return Response(status=status.HTTP_204_NO_CONTENT)
+
+    def destroy(self, request, *args, **kwargs):
+        """
+        Either delete a single or many model instances in bulk
+
+        DELETE /dogs/
+        {
+            "dog": {
```

Comment:
I'm not sure depending on the URL structure is safe, but the default [`destroy()`](https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/mixins.py#L92) method is pretty light. One option might be to try to do `self.get_object()` and if that raises, assume it's an unsupported request and return an error. By default it would just be an ISE, I think?